lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -53,13 +50,18 @@ export function contentItemForContent(ltiLaunchURL, content, extraParams) {
       break;
   }
 
+  const queryParams = new URLSearchParams();
+  Object.entries(params).forEach(([key, value]) =>
+    queryParams.append(key, value)
+  );
+
   return {
     '@context': 'http://purl.imsglobal.org/ctx/lti/v1/ContentItem',
     '@graph': [
       {
         '@type': 'LtiLinkItem',
         mediaType: 'application/vnd.ims.lti.v1.ltilink',
-        url: `${ltiLaunchURL}?${stringify(params)}`,
+        url: `${ltiLaunchURL}?${queryParams}`,
```

Comment:
Do you feel that this has sufficient test coverage? i.e. "You've tested this, right?"

Reply:
Yes. We've got tests that cover the full range of possible arguments to this function (different types of `content` and use of `extraParams`, arguments that require URI encoding). So I think we're good.